hack/devtools/local_dev_env.sh

```diff
@@ -7,6 +10,71 @@
 # The steps here are the ones defined in docs/deploy-development-rp.md
 # We recommend to use this script after you understand the steps of the process, not before.
 
+PLATFORM_WORKLOAD_IDENTITY_ROLE_SETS='[
+    {
+        "openShiftVersion": "4.14",
+        "platformWorkloadIdentityRoles": [
+            {
+                "operatorName": "CloudControllerManager",
+                "roleDefinitionName": "Azure RedHat OpenShift Cloud Controller Manager Role",
+                "roleDefinitionId": "/providers/Microsoft.Authorization/roleDefinitions/a1f96423-95ce-4224-ab27-4e3dc72facd4",
+                "serviceAccounts": ["openshift-cloud-controller-manager:cloud-controller-manager"]
+            },
+            {
+                "operatorName": "ClusterIngressOperator",
+                "roleDefinitionName": "Azure RedHat OpenShift Cluster Ingress Operator Role",
+                "roleDefinitionId": "/providers/Microsoft.Authorization/roleDefinitions/0336e1d3-7a87-462b-b6db-342b63f7802c",
+                "serviceAccounts": ["openshift-ingress-operator:ingress-operator"]
+            },
+            {
+                "operatorName": "MachineApiOperator",
+                "roleDefinitionName": "Azure RedHat OpenShift Machine API Operator Role",
+                "roleDefinitionId": "/providers/Microsoft.Authorization/roleDefinitions/0358943c-7e01-48ba-8889-02cc51d78637",
+                "serviceAccounts": ["openshift-machine-api:machine-api-operator"]
+            },
+            {
+                "operatorName": "StorageOperator",
+                "roleDefinitionName": "Azure RedHat OpenShift Storage Operator Role",
+                "roleDefinitionId": "/providers/Microsoft.Authorization/roleDefinitions/5b7237c5-45e1-49d6-bc18-a1f62f400748",
+                "serviceAccounts": [
+                    "openshift-cluster-csi-drivers:azure-disk-csi-driver-operator",
+                    "openshift-cluster-csi-drivers:azure-disk-csi-driver-controller-sa"
+                ]
+            },
+            {
+                "operatorName": "NetworkOperator",
+                "roleDefinitionName": "Azure RedHat OpenShift Network Operator Role",
```

Comment:
Nit - we made some corrections to the `roleDefinitionName` names in RP-Config, so we should make them here as well: https://msazure.visualstudio.com/AzureRedHatOpenShift/_git/RP-Config/pullrequest/10029612